src/vs/workbench/contrib/snippets/browser/commands/configureSnippets.ts

```diff
@@ -66,13 +73,19 @@ async function computePicks(snippetService: ISnippetsService, userDataProfileSer
 				}
 			}
 
+			let desc = names.size === 0
+				? nls.localize('global.scope', "(global)")
+				: nls.localize('global.1', "({0})", [...names].join(', '));
+			if (added.has(basename(file.location)) && source) {
+				desc = source + ' ' + desc;
+			}
+
 			existing.push({
 				label: basename(file.location),
 				filepath: file.location,
-				description: names.size === 0
-					? nls.localize('global.scope', "(global)")
-					: nls.localize('global.1', "({0})", [...names].join(', '))
+				description: desc
```

Comment:
An alternative that worth trying is to leave the description as-is and use the `detail` property here. It renderes on a separate line below the label, description pair and would have space for the file path

Reply:
![image](https://user-images.githubusercontent.com/17105034/207764051-be8d9371-a946-4e5b-8d70-2ac66e742644.png)
This is the case of replacing it with detail